content/material_and_questions.md

```diff
@@ -44,4 +53,151 @@ In the below table, any minimum or maximum variation between material years are
+Context is defined as the verses or verses that are required.
+
+##### Quote
+A single verse
+The Question Type abbreviation is "QT"
+
+##### Quote-these-Two-Verses
+Two consecutive verses
+The Question Type abbreviation is "Q2V"
+Should be verses that make more sense when quoted together.  Example: Gal 5:22-23 
+
+#### Finish
+Will be direct quotes from Scripture
+Must be quoted word-perfect
+Must be strong enough to stand on its own without requiring additional verses to explain it.
+Must be spiritually significant.
+The first five words must be unique to all Finish questions.
+The quizmaster will read the first five words of the question (less if a quizzer jumps)
+Quizzers may go back and correct their answers if done in the 30-second time limit.  The quizzer will be required to make one full correct rotation. However, the quizzer can be called correct if they have made a mistake only in one area of the verse AND correct the mistake making it abundantly clear to the quiz master and answer judge that they know where their mistake was, and what he corrected words are.
+Context is defined as the verses or verses that are required.
+
+##### Finish-the-Verse
+Begins at the beginning of a single verse and finishes at the end of that verse.
+The Question Type abbreviation is "FTV"
+
+##### Finish-This
+Begins in the middle of a verse and finishes at the end of the verse.  A FT must start at the beginning of a sentence or the beginning of a quotation.
+The Question Type abbreviation is "FT"
+
+##### Finish-these-Two-Verses
+Begins at the beginning of a verse and continues to the end of the next verse. Used when a verse is not strong enough on its own and needs another verse to explain it or can be used when two verses combined form an even stronger thought. 
+The Question Type abbreviation is "F2V"
+
+##### Finish-This-and-the-Next
+Begins in the middle of a verse and finishes at the end of the next verse.  A FTN must start at the beginning of a sentence or the beginning of a quotation. Used when part of a verse is not strong enough on its own and needs another verse (or part of a verse) to explain it or can be used when two verses combined form an even stronger thought.
+The Question Type abbreviation is "FTN"
+
+FTV questions should not start out with an insignificant introduction such as “He replied” or “He said to them”.  These verses, if spiritually significant, should be used as FT or FTN type questions.  For example, Luke 11:28 says, He replied, “Blessed rather are those who hear the word of God and obey it.”  This verse could be used as an excellent FT question starting with “Blessed rather are those who …”. 
```

Comment:
I don't know a good place to put. Right now it's under the FTN section. Ideally it goes under the Finish header, but at that point I haven't defined the sub-types yet.

Reply:
I think it works fine under the "FTN" section, honestly.

Reply:
WFM.